tap_twilio/streams.py

```diff
@@ -12,313 +12,125 @@
 #   bookmark_type: Data type for bookmark, integer or datetime
 #   children: A collection of child endpoints (where the endpoint path includes the parent id)
 #   parent: On each of the children, the singular stream name for parent element
-
+# todo review task_channels.json, review all fields with json information that needs extra work, TaskQueue Statistics
 STREAMS = {
     # Reference: https://www.twilio.com/docs/usage/api/account#read-multiple-account-resources
-    'accounts': {
```

Comment:
this is fully removing existing functionalities of the tap. Is that really that we want?

Can we extend the current functionalities somehow rather than deleting them completely?